internal_ws/yktrace/src/sir.rs

```diff
@@ -47,7 +48,7 @@ pub struct Sir<'m> {
 }
 
 impl<'m> Sir<'m> {
-    pub fn new(mmap: &'m Mmap) -> Result<Self, ()> {
+    pub fn new(mmap: &'m Mmap) -> Result<Self, Box<dyn Error>> {
```

Comment:
Can you also replace all unwraps with `?`?

Reply:
I think so, but I need to take a higher level view of the API to understand that. I think that can be done in a subsequent PR.